src/sdk/python/src/iotea/core/util/mqtt_client.py

```diff
@@ -47,8 +47,10 @@ def __init__(self, connection_string, topic_ns=os.environ.get('MQTT_TOPIC_NS', N
             self.logger = logging.getLogger(client_id)
 
         self.client = CustomMqttClient(client_id, on_reconnect=self.__on_reconnect)
+        self.connection_string = connection_string
         self.client.config['reconnect_retries'] = 100000
```

Comment:
should we move such config outside? (can be done in a upcoming PR)

Reply:
Moved magic numbers out into module constants (for local configuration options) or into constants.py module for global configuration options.